WTWSMS/common/landed_titles/mercenaries.txt

```diff
@@ -2565,4 +2564,80 @@ d_ghanan_band = {
 	strength_growth_per_century = 1.0
```

Comment:
> I didn't know that I was quoting Margeret Thatcher: "consensus is the absence of leadership" has long been one of my life's mottoes. You talked about "concertation", which seems to be a synonym of "conensus".

You are quoting the Iron Lady, which makes it particularly ridiculous. The two alternatives you yourself present here is that either nothing is done or I decide things unilaterally. Mod development works through discussion when it is collaborative. I'm not going to send troops to defend an invasion of the Falklands, sell out welfare or have access to nuclear codes. In fact, I'm not even the lead developer or Prime Minister. 😛